sound/soc/sof/ipc4-loader.c

```diff
@@ -151,6 +151,23 @@ static size_t sof_ipc4_fw_parse_basefw_ext_man(struct snd_sof_dev *sdev)
 	return payload_offset;
 }
 
+struct sof_ipc4_fw_module *sof_ipc4_find_module_by_uuid(struct snd_sof_dev *sdev,
+							const guid_t *uuid)
+{
+	struct sof_ipc4_fw_data *ipc4_data = sdev->private;
+	struct sof_ipc4_fw_library *fw_lib;
+	int i, id;
+
+	idr_for_each_entry(&ipc4_data->fw_lib_idr, fw_lib, id) {
```

Comment:
s/id/lib_id/  for clarity? I really hate the mixed use of i and id...

Reply:
OK, I can do that in similar places.